install-dev/init.php

```diff
@@ -50,7 +50,9 @@
 
 require_once _PS_CORE_DIR_.'/config/defines.inc.php';
 require_once _PS_CORE_DIR_.'/config/autoload.php';
-require_once _PS_CORE_DIR_.'/config/bootstrap.php';
+if (file_exists(_PS_CORE_DIR_.'/app/config/parameters.yml')) {
```

Comment:
We have a duplicate here. In the file `/config/bootstrap.php`, we already test the file parameters.yml exists.

```php
$fmtParamYml = (int)@filemtime(__DIR__. '/../app/config/parameters.yml');
```